yml/cdb_latiss.yaml

```diff
@@ -0,0 +1,829 @@
+---
+name: cdb_latiss
+"@id": "#cdb_latiss"
+description: Consolidated Database for LATISS
+tables:
+- name: Exposure
+  "@id": "#Exposure"
+  primaryKey:
+  - "#Exposure.day_obs"
+  - "#Exposure.seq_num"
+  constraints:
+  - name: un_exposure_id
+    "@id": "#Exposure.un_exposure_id"
+    "@type": Unique
+    description: Ensure exposure_id is unique.
+    columns:
+    - "#Exposure.exposure_id"
+  columns:
+  - name: exposure_id
+    "@id": "#Exposure.exposure_id"
+    datatype: long
+    nullable: false
+    description: Unique identifier.
+  - name: exposure_name
+    "@id": "#Exposure.exposure_name"
+    datatype: string
+    length: 20
+    nullable: false
+    description: Official name of the exposure.
+  - name: controller
+    "@id": "#Exposure.controller"
+    datatype: string
+    length: 1
+    nullable: false
+    description: The abbreviation of the controller used for the observation (O, C)
+  - name: day_obs
+    "@id": "#Exposure.day_obs"
+    datatype: long
+    nullable: false
+    description: Day of observation
+  - name: seq_num
+    "@id": "#Exposure.seq_num"
+    datatype: long
+    nullable: false
+    description: Sequence number
+  - name: physical_filter
+    "@id": "#Exposure.physical_filter"
+    datatype: string
+    length: 64
+    description: ID of physical filter, the filter associated with a particular instrument.
+  - name: band
+    "@id": "#Exposure.band"
+    datatype: string
+    length: 32
+    description: Name of the band used to take the exposure where this source was
+      measured. Abstract filter that is not associated with a particular instrument.
+  - name: ra
```

Comment:
I think we should consider using ObsCore names for columns that have those roles in this schema.  So this one would be `s_ra`.  In that case you can also set `tap:std` to 1.

The standard metadata should also be included: UCD `pos.eq.ra` and units `deg`.